BetterTeams/src/main/java/com/booksaw/betterTeams/integrations/placeholder/TeamPlaceholders.java

```diff
@@ -91,6 +92,37 @@ public String onPlaceholderRequest(Player player, @NotNull String identifier) {
 		return TeamPlaceholderService.getPlaceholder(identifier, team, tp);
 	}
 
+	@Override
+	public @Nullable String onRequest(OfflinePlayer player, @NotNull String identifier)  {
```

Comment:
Could this method be merged somehow (i.e. have a shared child method) with the method above to avoid duplication please? :)